podium/talks/views.py

```diff
@@ -26,5 +27,12 @@ def session_list_view(request):
     return render(request, 'talks/sessions.html', context)
 
 
-def session_talk_list_view(request):
-    pass
+def session_talk_list_view(request, id):
+    try:
```

Comment:
Check out `get_object_or_404` for a handy shortcut to this pattern:
https://docs.djangoproject.com/en/1.11/topics/http/shortcuts/#get-object-or-404